Sources/IO/Geometry/IFCImporter/index.js

```diff
@@ -94,11 +94,34 @@ function vtkIFCImporter(publicAPI, model) {
         .buildFromRadian()
         .setMatrix(userMatrix);
 
+      const rotMatrix = [
+        userMatrix[0],
+        userMatrix[1],
+        userMatrix[2],
+        0,
+        userMatrix[4],
+        userMatrix[5],
+        userMatrix[6],
+        0,
+        userMatrix[8],
+        userMatrix[9],
+        userMatrix[10],
+        0,
+        0,
+        0,
+        0,
+        1,
+      ];
+      const rotationMatrix = vtkMatrixBuilder
+        .buildFromRadian()
+        .setMatrix(rotMatrix);
```

Comment:
Could the following work ?
`const rotationMatrix = vtkMatrixBuilder.buildFromRadian().multiply3x3(mat3.fromMat4(userMatrix))`

Reply:
It works with a small adjustment